### ISSUE TITLE
Confirmation Link mehrmals klicken

### ISSUE THREAD

**Issue by annakress** [initiatived21/d21#147]
Wenn ich meine Email schon bestätigt habe, und noch mal bestätige, komme ich auf eine recht häßliche Seite. Müssen wir überarbeiten:

![resend](https://cloud.githubusercontent.com/assets/2988567/17904167/8fb7224a-696e-11e6-91d0-b3aa17e590e8.png)

**Comment by marcaltmann** [comment]
Achso, ich seh gerade, dass das Issue schon älter ist. Inzwischen ist die Seite gestyled. Siehe Screenshot. Allerdings gibt es immer noch eine Devise-Fehlermeldung (benutzer konnte nicht gespeichert werden), die vielleicht etwas irreführend ist.
![devise](https://cloud.githubusercontent.com/assets/2727647/18513152/8932a532-7a8c-11e6-9390-f57731eee8f8.png)

**Comment by annakress** [comment]
Ja, die Meldung "benutzer konnte..." muss auf jeden Fall weg. Ausserdem sieht es so aus, als wäre das Label E-Mail verrutscht? Direkt unter dem Listenpunkt steht noch einmal Email.
